web/layout_attributes/using_layout_attributes/my_horizontal_element.html

```diff
@@ -0,0 +1,12 @@
+<link rel="import" href="packages/polymer/polymer.html">
+<dom-module id="my-horizontal-element">
+  <template>
+    <section class="layout horizontal">
```

Comment:
I changed these to use `@apply(--layout-xxx)`. I assume this is the new way to do it.